django_spanner/__init__.py

```diff
@@ -46,7 +46,7 @@
 USE_EMULATOR = os.getenv("SPANNER_EMULATOR_HOST") is not None
 
 # Only active LTS django versions (2.2.*, 3.2.*) are supported by this library right now.
```

Comment:
nit: (2.2.*, 3.2.*, 4.2.*)

Reply:
Thanks for the comments but this is just a draft PR that I am using for testing for Django 4.2 so its just a hack to test it and not ready for review.

Reply:
Thank you @ankiaga , if the effort to support 5.0 is not a lot on top of 4.2, can we also add support to 5.0?
This will allow spanner users to upgrade to django 5.0, everybody is on 3.2 currently, I prefer migrate from 3.2->5.0 in one go, instead of 3.2 -> 4.2 -> then in 2 years -> 5.2

Reply:
We are first planning to release 4.2 and then we will take a call for supporting 5.0 as well but we are already discussing this internally

Reply:
With this release, could it at least be possible to override the version check? I'm fairly certain these changes will work for 5.

Reply:
+1 to @grjones , if we don't officially support 5.0, maybe allow users to bypass the check and take our own risk

Reply:
Please note that these are not complete changes for Django 4.2 and I am fixing them as I am finding more issues.

Also to release 5.0 (similar to 4.2) we need to make sure that all the system tests from Django core repo works whose number is in tens of thousands. This is the main challenge to release a new version. Currently many hundreds of tests are failing as I am trying to migrate to 4.2 and root causing and fixing them takes a lot of time